kong/api/endpoints.lua

```diff
@@ -303,12 +303,17 @@ end
 local function get_collection_endpoint(schema, foreign_schema, foreign_field_name, method)
   return not foreign_schema and function(self, db, helpers)
     local next_page_tags = ""
+    local next_page_size = ""
 
     local args = self.args.uri
     if args.tags then
       next_page_tags = "&tags=" .. escape_uri(type(args.tags) == "table" and args.tags[1] or args.tags)
     end
 
+    if (type(args.size) == "number") then
```

Comment:
style: this `if` condition should not use `(...)`.

Reply:
wow, ok, removed :smile: